commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/CreateServerSSLContext.java

```diff
@@ -62,6 +63,10 @@
     // Multiple usage
     private final String algorithm;         // keystore manager, truststore manager
 
+    // Default set of cipher suites for TLSv1.3 to be set in 'cipher-suite-names' attribute.
+    public static final String CIPHER_SUITE_NAMES_VALUE =
```

Comment:
I wonder if we could find some better name for this constant, for example `TLS13_CIPHER_SUITE_NAMES` ?

Reply:
Okay, makes sense, I've just updated. Thanks for this! :)